src/NodeManipulator/PropertyManipulator.php

```diff
@@ -39,13 +39,23 @@
     /**
      * @var string[]|class-string<Table>[]
      */
-    private const DOCTRINE_PROPERTY_ANNOTATIONS = [
+    private const ALLOWED_NOT_READONLY_CLASS_ANNOTATIONS = [
+        'ApiPlatform\Core\Annotation\ApiResource',
+        'ApiPlatform\Metadata\ApiResource',
         'Doctrine\ORM\Mapping\Entity',
         'Doctrine\ORM\Mapping\Table',
         'Doctrine\ORM\Mapping\MappedSuperclass',
         'Doctrine\ORM\Mapping\Embeddable',
     ];
 
+    /**
+     * @var string[]|class-string<Table>[]
+     */
+    private const ALLOWED_NOT_READONLY_CLASS_ATTRIBUTES = [
+        'ApiPlatform\Metadata\ApiResource',
```

Comment:
this `ApiPlatform\Metadata\ApiResource`  duplicated with already registered on constant `ALLOWED_NOT_READONLY_CLASS_ANNOTATIONS`, I think single constant is fine for both annotation and attribute verification.

Reply:
Oh, thanks. Fixed.